### PR TITLE
Neurotoxic Spit Tweaks.

### PR DESCRIPTION
**Pull request by vilereaver41**
## About The Pull Request

Tweaks neurotoxic spit & the gas it leaves to actually be helpful and effective.

## Why It's Good For The Game

Marines currently are nearly unstoppable unless your entire hive knows what they are doing and are skilled which isn't 24/7. current neuro spits are worthless and easily fixed, marines have so many tools that help crush forward while  Xenos has no real strength to stop other than acid turrets which are only temporary, more so as they cant be built as close to each other anymore.

This tweak allows neurotoxic spit for sentinel, praetorian, queen to be worth something in team battles that slow a marines push and makes them weary rather than having them continue forward unchecked which is hard to deal with. in return I've increased the cost somewhat so this isn't something that can be spammed but used well to assist your team in stopping a marine push as well as pushing forward as well. 

Accounting for spit damage, increased plasma cost, gas strength, this would increase each caste general Neurospit/splash by 30%

## Changelog
:cl:
balance: neurotoxic spit for Sentinel, Praetorian,
balance: After extensive testing the values have been increased slightly and should now be the best they can be without gameplay being unfair.
/:cl:

### CODE REVIEW

File: code/modules/projectiles/ammo_datums.dm [hunk 1]
```diff
@@ -2079,7 +2078,7 @@ datum/ammo/bullet/revolver/tp44
 	name = "flare"
 	ping = null //no bounce off.
 	damage_type = BURN
-	flags_ammo_behavior = AMMO_INCENDIARY
+	flags_ammo_behavior = AMMO_EXPLOSIVE|AMMO_INCENDIARY
```
Comment (BraveMole) [inline code/modules/projectiles/ammo_datums.dm:2081]: ???

**Review by BraveMole — CHANGES_REQUESTED** [review]

File: code/modules/projectiles/ammo_datums.dm (hunk 1 above)
Comment (vilereaver41) [reply code/modules/projectiles/ammo_datums.dm:2081]: Ehh wierd i never changed that, its from a fresh master

File: code/modules/projectiles/ammo_datums.dm [hunk 2]
```diff
@@ -1695,58 +1695,57 @@ datum/ammo/bullet/revolver/tp44
...
 /datum/ammo/xeno/toxin/heavy/upgrade2
-	smoke_strength = 0.75
-	reagent_transfer_amount = 9.5
+	smoke_strength = 0.9
+	reagent_transfer_amount = 10.5
 
 /datum/ammo/xeno/toxin/heavy/upgrade3
-	smoke_strength = 0.8
-	reagent_transfer_amount = 10
+	smoke_strength = 0.10
```
Comment (Ragecrabbo) [inline code/modules/projectiles/ammo_datums.dm:1747]: probably should have this be 1.0 instead of 0.10

**Comment by KornFlaks** [comment]
Was that last review by Ragecrabbo intended to be resolved with nothing? Also dont resolve your own PRs reviews.

File: code/modules/projectiles/ammo_datums.dm (hunk 2 above)
Comment (TiviPlus) [reply code/modules/projectiles/ammo_datums.dm:1747]: ^^

File: code/modules/projectiles/ammo_datums.dm [hunk 3]
```diff
@@ -1621,19 +1621,19 @@ datum/ammo/bullet/revolver/tp44
...
+	added_spit_delay = 3
 	damage_type = STAMINA
 	accurate_range = 5
 	max_range = 10
 	accuracy_var_low = 3
 	accuracy_var_high = 3
-	damage = 30
-	stagger_stacks = 1
-	slowdown_stacks = 1
+	damage = 450
```
Comment (Pariah919) [inline code/modules/projectiles/ammo_datums.dm:1631]: why is this 450 now

**Review by SplinterGP — COMMENTED** [review]
this will be needed more testing because its a fairly big buff on some stuff, im particularly weary of the damage increases on spit because after stamina is gone it does 100% of that damage as direct damage, however i think its not of all bad, also i see its affected by armor so its not thaaat bad

**Comment by vilereaver41** [comment]
I closed it why was it reopened

**Comment by KornFlaks** [comment]
> balance: neurotoxic spit for Sentinel, Praetorian, **Queen** to be stronger to a fair degree
Hahahaha... *sob*.

**Comment by David-Stormwell** [comment]
Requires further testing, in of itself the idea of a buff doesn't sound bad, but if it's overdone may lead to huge powercreep stalling games quite some, perhaps making a lowely T1 as significant as a T2, check with player input on how to adjust this, but the idea of this PR I approve.

**Comment by BraveMole** [comment]
Fix your CL, queen doesn't have neuro

**Review by vilereaver41 — COMMENTED** [review]
> Fix your CL, queen doesn't have neuro

done

File: code/modules/projectiles/ammo_datums.dm [hunk 4]
```diff
@@ -1695,58 +1695,55 @@ datum/ammo/bullet/revolver/tp44
...
 
 /datum/ammo/xeno/toxin/upgrade3
-	smoke_strength = 0.65
-	reagent_transfer_amount = 8
-
-
-/datum/ammo/xeno/toxin/medium //Queen
+	smoke_strength = 0.8
+	reagent_transfer_amount = 10
+	/datum/ammo/xeno/toxin/medium //Queen
```
Comment (BraveMole) [inline code/modules/projectiles/ammo_datums.dm:1708]:
```suggestion
/datum/ammo/xeno/toxin/medium //Queen
```

**Review by BraveMole — CHANGES_REQUESTED** [review]

**Review by BraveMole — APPROVED** [review]

**Pull request merged**